config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.2/README.md

```diff
@@ -0,0 +1,104 @@
+
+For full release notes, compatiblity, installation, and user information, see the [IBM block storage CSI driver documentation](https://www.ibm.com/docs/en/stg-block-csi-driver/1.11.1).
+
+Supported IBM storage systems:
+  - IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx))
+  - IBM FlashSystem A9000 and A9000R
+  - IBM DS8000 Family
+  
+## Prerequisites
+
+Review the [compatibility and requirements documentation](https://www.ibm.com/docs/en/stg-block-csi-driver/1.11.1?topic=installing-compatibility-requirements). 
+
+**Important:** Be sure to complete all prerequisite and installation steps before assigning hosts to your location. Do not create a Kubernetes cluster. This is done through Satellite.
+
+## IBM block storage CSI driver parameters and how to retrieve them
+Retrieve all parameters required by this template.
+
+```sh
+ibmcloud sat storage template get --name ibm-system-storage-block-csi-driver --version 1.11.2
+```
+
+ **IBM block storage CSI driver parameters**
+ 
+ For more information about the fields and examples, review the [Creating a StorageClass documentation](https://www.ibm.com/docs/en/stg-block-csi-driver/1.11.1?topic=configuring-creating-storageclass)
+| Parameter | Required? | Description | Default value if not provided |
+| --- | --- | --- | --- |
+| `namespace` | Optional | The namespace where you want to create the deployment. | `default` |
+| `name` | Required | The name of the storage class name that is created. | N/A |
+| `space-efficiency` | Optional | The space efficiency of the volume that is created. | N/A |
+| `pool` | Required | Name of an existing pool on the storage system where you want to create the volume. | N/A |
+| `secret-name` | Required | The name of your existing Kubernetes secret. | N/A |
+| `secret-namespace` | Required | The namespace that your Kubernetes secret is in. (can be different then the deployment namespace) | N/A |
+| `fstype` | Optional | The file system type. | `ext4` |
+| `prefix` | Optional | The prefix name of the volume that is created. | N/A |
+| `VolumeExpansion` | Optional |  Specify `true` to allow volume expansion or `false` to disallow volume expansion. | `false` |
+
+
+## Creating the IBM block storage CSI driver storage configuration
+
+**Example `sat storage config create` command**
+
+```sh
+ibmcloud sat storage config create --name <config-name> --template-name ibm-system-storage-block-csi-driver --template-version 1.11.2 --location <location-name/location-id> -p "namespace=<namespace>" -p "name=<sc-name>" -p "space-efficiency=<space-efficiency>" -p "pool=<pool>" -p "secret-name=<secret-name>" -p "secret-namespace=<secret-namespace>" -p "fstype=<fstype>" -p "prefix=<prefix>" -p "VolumeExpansion=<VolumeExpansion>"
```

Comment:
This command for creating config using this template is incorrect.
Please pass correct parameters list to this command, specified in `custom-parameters.json`.
Also please fix the same command in older version 1.11.1 README.

After creating the config, we can add new section to add a user defined storageclass and specify following command:
`ibmcloud sat storage config class add --config-name NAME --name NAME --param PARAM [--param PARAM ...]`
Here PARAMS are the parameters list defined in `storage-class-parameters.json`